packages/back-end/generated/spec.yaml

```diff
@@ -1001,6 +1007,199 @@ paths:
+        '200':
+          content:
+            application/json:
+              schema:
+                type: object
+                required:
+                  - savedGroup
+                properties:
+                  savedGroup:
+                    $ref: '#/components/schemas/SavedGroup'
+    put:
+      parameters:
+        - $ref: '#/components/parameters/id'
+      tags:
+        - saved-groups
+      summary: Update a single saved group
+      operationId: putSavedGroup
+      x-codeSamples:
+        - lang: cURL
+          source: |
+            curl -X PUT https://api.growthbook.io/api/v1/saved-groups/grp_123abc \
+              -H "Content-Type: application/json" \
+              -u secret_abc123DEF456: \
+              -d '{"name": "interal-users", "values": "userId-123, userId-345, userId-678"}'
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              type: object
+              required:
+                - name
+                - values
+                - owner
+              properties:
+                name:
+                  type: string
+                  description: The display name of the Saved Group
+                values:
+                  type: array
+                  items:
+                    type: string
+                  description: 'An array of values to target (Ex: a list of userId''s).'
+                owner:
+                  type: string
+                  description: 'The name of the user in your GrowthBook account that owns this Saved Group. If no owner, you can pass in an empty string.'
+      responses:
+        '200':
+          content:
+            application/json:
+              schema:
+                type: object
+                required:
+                  - savedGroup
+                properties:
+                  savedGroup:
+                    $ref: '#/components/schemas/SavedGroup'
+    patch:
+      summary: Partially update a single saved group
+      parameters:
+        - $ref: '#/components/parameters/id'
+      tags:
+        - saved-groups
+      operationId: patchSavedGroup
+      x-codeSamples:
+        - lang: cURL
+          source: |
+            curl -X PATCH https://api.growthbook.io/api/v1/saved-groups/grp_123abc \
+              -H "Content-Type: application/json" \
+              -u secret_abc123DEF456: \
+              -d '{"values": [ "userId-123", "userId-345" ]}'
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              type: object
+              required: []
+              properties:
+                name:
+                  type: string
+                  description: The display name of the Saved Group
+                values:
+                  type: array
+                  items:
+                    type: string
+                  description: 'An array of values to target (Ex: a list of userId''s).'
```

Comment:
Looks like there's something funny going on with the small quotes here. Unless this is YAML magic for escaping quotes? I realize this is in the generated one so maybe it's valid.

Reply:
Yea - that is odd. It renders in the docs correctly, FWIW.

<img width="423" alt="Screen Shot 2023-04-18 at 6 45 11 AM" src="https://user-images.githubusercontent.com/75274610/232754172-859ab25c-27b3-49c9-a7c4-f93ddf2a5b8e.png">

However, in this case, the apostrophe in `userId's` doesn't actually need to be there, so I've removed it.